tslint.json

```diff
@@ -10,7 +10,45 @@
 			true,
 			"always"
 		],
-		"triple-equals": true
+		"triple-equals": true,
+		"no-banned-terms": true,
+		"no-delete-expression": true,
+		"no-document-domain": true,
+		"no-disable-auto-sanitization": true,
+		"no-duplicate-parameter-names": true,
+		"no-exec-script": true,
+		"no-function-constructor-with-string-args": true,
+		"no-octal-literal": true,
+		"no-reserved-keywords": true,
+		"no-string-based-set-immediate": true,
+		"no-string-based-set-interval": true,
+		"no-string-based-set-timeout": true,
+		"no-eval": true,
+		"quotemark": false,
+		"no-relative-imports": false,
+		"max-line-length": [
+			true,
+			{
+			  "limit": 200,
+			  "ignore-pattern": "^import [^,]+ from |^export | implements"
+			}
+		  ],
+		"trailing-comma": false,
+		"one-line": false,
+		"ordered-imports": [
+			true,
+			{
+			  "import-sources-order": "case-insensitive",
+			  "named-imports-order": "case-insensitive"
+			}
+		],
+		"no-constant-condition": [
+			true,
+			{
+				"checkLoops": false
+			}
+		]
 	},
-	"defaultSeverity": "warning"
+	"defaultSeverity": "warning",
+	"extends": ["tslint-microsoft-contrib/recommended", "tslint:recommended"]
```

Comment:
if we specify "tslint-microsoft-contrib/recommended" in extends here, 'do we also need to specify rulesDirectory in line no 2? what is the different between the two?

Reply:
Line no 2, specifies the rules directory, but it doesn't say which rules to use from the directory.
Extends specifies which rule config files to use and tool will search into rules directory.